README.md

```diff
@@ -34,7 +34,7 @@ This sample project illustrates 2 usage scenarios of Karma integration in the Gu
 
 While there exist integration of the karma-runner into [Grunt](http://gruntjs.com/) based builds (https://github.com/karma-runner/grunt-karma)
 we've decided _not_ to create a dedicated plugin for Gulp.js. The reasoning here is that it is very, very easy to
-invoke Karma's [public API](http://karma-runner.github.io/0.12/dev/public-api.html) directly from a Gulp.js task, as illustrated in this repository. Writing a dedicated plugin
+invoke Karma's [public API](http://karma-runner.github.io/0.13/dev/public-api.html) directly from a Gulp.js task, as illustrated in this repository. Writing a dedicated plugin
```

Comment:
Let's use `/latest/dev` so we don't have to update it the next time around again :)